pkg/beaconfqdn/dissector.go

```diff
@@ -95,7 +191,7 @@ func (d *dissector) start() {
 					"icerts": bson.M{"$anyElementTrue": []interface{}{"$dat.icerts"}},
 				}},
 				{"$group": bson.M{
-					"_id":    "$src",
+					"_id":    bson.M{"src": "$src", "uuid": "$src_network_uuid", "network": "$src_network_name"},
```

Comment:
We cannot group on network name as multiple network names may be associated with the same network uuid.
This is a downstream result of how winlogbeat identifies Sysmon agents.

Instead, please use `"network_name": bson.M{"$first": "$src_network_name"}`. Unfortunately, this will need to be added to most of the clauses below.

Reply:
In the past, queries used `$last` to group on network uuids so that we'd be referencing the last known netbios name in case it ever changed. Is there a reason why we should use `$first` here instead?

Reply:
Nope. Great catch. I'm just a big dummy. :clown_face: Please use `$last`.